devicetypes/smartthings/leedarson-dim-bulb.src/leedarson-dim-bulb.groovy

```diff
@@ -0,0 +1,149 @@
+/**
+*  Copyright 2017 SmartThings
+*
+*  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License. You may obtain a copy of the License at:
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+*  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+*  for the specific language governing permissions and limitations under the License.
+*
+*  Author: SmartThings
+*  Date: 2016-01-19
+*
+*  This DTH should serve as the generic DTH to handle RGBW ZigBee HA devices
+*/
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition (name: "ZigBee DIM Bulb", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", runLocally: false, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true,mnmn: "SmartThings",vid: "generic-dimmer") {
+
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Switch"
+        capability "Switch Level"
+        capability "Health Check"
+        capability "Light"
+
+        fingerprint profileId: "0104", inClusters: "0000, 0004, 0003, 0006, 0008, 0005, FFFF, 1000", outClusters: "0019", model: "ZBT-DIMLight-GLS0000", manufacturer: "LDS", deviceJoinName: "Bulb"
```

Comment:
This looks like a standard dimmable bulb so you can use the Zigbee Dimmer DTH and don't need to create a new one.